app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,28 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+    @user = current_user
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author_id = current_user.id
+
```

Comment:
as requested by the previous reviewer a post is created on behalf of the currently logged-in user 😄 . I know it might be annoying but getting this little task done will go a long way when learning in future. 👍🏾

Reply:
I'm confusing about "It will return the first user from the database."  In this project requirement it says current_user should be the first user in database. I can change it to logged in user but my time is very limited now. I do not want to repeat whole week because of this misunderstood :D

Reply:
Okay, now the posts are created on that particular user.

Reply:
the first user is the current_user

Reply:
Did I do it correctly?